docs/api/security/psa_internal_storage.md

```diff
@@ -0,0 +1,20 @@
+## PSA Internal Storage
+
+### Description
+
+PSA internal storage APIs allows saving and retrieving data from PSA internal flash.
+
+PSA internal storage is implementation varies depending on the target type:
+- on single core ARMv7-M target it PSA internal storage APIs are implemented by calling to "default" internal TDBStore instance.
```

Comment:
on single core ARMv7-M target it PSA internal storage APIs are implemented by calling to "default" internal TDBStore instance allocated by the KVStore configuration (see ../../reference/configuration/Storage.md)